test/manual/amp-analytics/analytics-fcp.html

```diff
@@ -0,0 +1,172 @@
+</p>
+<amp-video
+    id="myVideo"
+    src="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerJoyrides.mp4"
+    width="358"
+    height="204"
+    layout="responsive"
+    controls
+    autoplay
+  >
+  <div placeholder>
+    This is a placeholder
+  </div>
+  <div fallback>
+    This is a fallback
+  </div>
+</amp-video>
+<button on="tap:myVideo.pause">Pause</button>
+<p>
+Donec pharetra molestie sollicitudin. Duis mattis eleifend rutrum. Quisque luctus tincidunt lacus, vitae lobortis nisi malesuada ac. Aliquam mattis leo vel elit rutrum, nec consequat massa vestibulum. Maecenas bibendum metus nec ante feugiat, eu faucibus orci mattis. Cras tristique sem non elit congue malesuada. Proin ornare, lacus et porttitor consequat, sapien urna rutrum diam, ac pellentesque ligula est eget nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Donec ultrices sollicitudin eros a placerat. Proin eget pulvinar est. Donec posuere ultrices odio at ultrices. Suspendisse potenti. Phasellus id orci id purus porttitor consectetur a at erat. Nullam volutpat ultricies nisl id maximus. Morbi porta ex ante, et egestas odio ultricies consequat.
+</p>
+<p>
+<ul>
+<li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+<li>Aliquam in ex porta, imperdiet elit sit amet, condimentum diam.</li>
+<li>Etiam fermentum nisi at porta pulvinar.</li>
+</ul>
+</p>
+<p>
+<ul>
+<li>Proin mattis neque vel elit posuere molestie.</li>
+<li>Integer tincidunt sem sed nunc auctor elementum.</li>
+<li>Integer a felis in ipsum aliquet auctor sit amet a neque.</li>
+</ul>
+</p>
+<p>
+<ul>
+<li>Sed suscipit dolor molestie, rhoncus quam ac, lacinia ex.</li>
+<li>Curabitur et tellus vel justo ultrices aliquet sed id turpis.</li>
+<li>Nam finibus risus at justo elementum bibendum.</li>
+<li>In non lacus non urna congue feugiat at vel diam.</li>
+</ul>
+</p>
+<p>
+<ul>
+<li>Integer hendrerit augue interdum dui venenatis, sit amet tristique mauris cursus.</li>
+<li>Etiam quis eros viverra, tincidunt justo in, facilisis nunc.</li>
+<li>Aliquam at lacus faucibus, congue lorem interdum, semper mauris.</li>
+<li>Ut vulputate erat vel feugiat pharetra.</li>
+<li>Morbi id augue id orci sagittis tempus.</li>
+<li>Vestibulum varius libero ac dignissim sodales.</li>
+</ul>
+</p>
+<p>
+<ul>
+<li>Aenean ac sem eget libero varius viverra sit amet vitae nunc.</li>
+</ul>
+</p>
+<amp-anim src="https://www.google.com/logos/doodles/2016/2016-doodle-fruit-games-day-14-5645577527230464-hp.gif" id="anim-id" width="665" height="220" layout="responsive">
+  <amp-img placeholder src="https://www.google.com/logos/doodles/2016/2016-doodle-fruit-games-day-14-5645577527230464.2-scta.png" width="665" height="220" layout="responsive"></amp-img>
+  <amp-analytics id="nestedAnalytics">
```

Comment:
To only test one behavior, you can remove the analytics component here. But since it's manual test file for your own usage, feel free to leave as it is.